### ISSUE TITLE
Blank page after login

### ISSUE THREAD

**Issue by Gambler13** [hellofresh/janus-dashboard#302]
I tried to setup Janus with the dashboard.
I used a custom Docker compose

```
version: '3.3'
services:
  janus:
    image: quay.io/hellofresh/janus
    ports:
      - "8080:8080"
      - "8081:8081"
    depends_on:
      - janus-database
    volumes:
      - ./janus.toml:/etc/janus/janus.toml
  janus-dashboard:
    image: quay.io/hellofresh/janus-dashboard
    ports:
      - "8082:80"
    depends_on:
      - janus-database
  janus-database:
      container_name: janus-database
      image: mongo
      ports:
        - "27017:27017"

```

After `docker-compose up` I'm able to navigate to the login page. But after the login the page stays blank. Any ideas?

**Comment by TechMaster** [comment]
I experienced same problem "blank page after login successfully into Janus Dashboard". I had run
```
docker-compose -f tests/docker-compose.test.yml up -d
```

**Comment by s4nji** [comment]
This should be fixed by 93f8a11fb3f07527803154542eaab77ff152c792; closing the issue.
